### DISCUSSION TITLE
Livestreaming for hours instead of seconds?

### DISCUSSION THREAD

**Discussion by TheBotlyNoob** [scivision/PyLivestream#103] [Q&A] [answered]
I need to use hours. But because the python library only supports seconds I have to multiply the number of hours by 3600 to get the number of seconds. But then ffmpeg throws:
`Invalid duration specification for t:`
And tons of 1's.

btw my code is:

```python
import pylivestream.api as pls
def livestream(hours):
    hours *= 3600
    pls.stream_screen(r"path\to\ini\pylivestream.ini", "youtube", assume_yes = True, timeout = hours)
```

**Comment by scivision** [comment, marked as answer]
You don't have to specify timeout=.  It might be a limitation of FFmpeg.  So if you can, just omit the timeout= parameter
